tests/test_registration_embargoes.py

```diff
@@ -617,14 +618,14 @@ def setUp(self):
         })
 
     @mock.patch('framework.tasks.handlers.enqueue_task')
-    def test_POST_register_make_public_immediately_creates_registration_approval(self, mock_enqueue):
+    def test_register_draft_without_embargo_creates_registration_approval(self, mock_enqueue):
         res = self.app.post(
-            self.project.api_url_for('node_register_template_page_post', template=u'Open-Ended_Registration'),
+            self.project.api_url_for('register_draft_registration', draft_id=self.draft._id),
             self.valid_make_public_payload,
             content_type='application/json',
             auth=self.user.auth
         )
-        assert_equal(res.status_code, 201)
+        assert_equal(res.status_code, 202)
 
         registration = Node.find().sort('-registered_date')[0]
```

Comment:
The implementation is such that a registration is created right away. We return 202 because that registration is not visible right away.